### ISSUE TITLE
Tnt will break but not drop blocks randomly

### ISSUE THREAD

**Issue by cappuccinocosmico** [Daenningas/EnchantedSword#25]
Its the same behavior as documented in: https://github.com/PaperMC/Paper/issues/4302 

Video evidence not shared to prevent cord leakage.

**Comment by jprknight** [comment]
https://github.com/PaperMC/Paper/blob/master/Spigot-Server-Patches/0016-Drop-falling-block-and-tnt-entities-at-the-specified.patch

Possible link here. I'll see if I can find what build this lands in.

**Comment by Daenningas** [comment]
Possible connection here.
https://github.com/PaperMC/Paper/blob/master/Spigot-Server-Patches/0016-Drop-falling-block-and-tnt-entities-at-the-specified.patch
Which went into this commit https://github.com/PaperMC/Paper/commit/298c9022b03cc5f8ba2d1a0cc3cac860a9d5757b
All September 19th.

Is this something you can repro now? We just updated to Papermc 246 which likely includes this commit.

**Comment by cappuccinocosmico** [comment]
Yeah it is still a problem, I just reproduced it this morning, although intrestingly enough it seems like the chance of it dropping items depends how frequently the tnt drops and how many blocks are destroyed. Its also an all or nothing thing, sometimes it will drop all of the items and other times it will drop none.
